__tests__/db.test.js

```diff
@@ -3,6 +3,7 @@ const db = require("../db/connection");
 const testData = require("../db/data/test-data/index");
 const seed = require("../db/seeds/seed");
 const app = require("../db/app");
+const { fetchEndpoint } = require("../db/models/model");
```

Comment:
File could just be required directly instead of running a function as well to do the same thing.